### DISCUSSION TITLE
wiki ignores plantuml settings and renders pages under edit via plantuml.requarks.io

### DISCUSSION THREAD

**Discussion by openpipe0** [requarks/wiki#4511] [Error / Bug Report] [answered]
![image](https://user-images.githubusercontent.com/68401158/135025480-1dbf7143-d014-4f71-add7-3107e54bf8d9.png)

![image](https://user-images.githubusercontent.com/68401158/135025643-b48e4c6f-f87e-44a9-bf7a-d3a56cd8da9a.png)

**Comment by NGPixel** [comment, marked as answer]
This is a limitation of how the client-side editor use modules. It will be addressed in v3.

Your diagrams are still rendered on your server when you click save and viewed afterwards.

**Reply by ethereal-engineer** [reply]
Ah. This would be why editing diagrams work but save and view does not.

**Comment by mulder999** [comment]
Is there a quick/nice way to disable preview meanwhile to avoid leaking any plant schema to the outside ?

As external workaround, thinking custom DNS entries or iptables redirection.
